src/job/models/ingestion/seedingJobCreator.ts

```diff
@@ -98,14 +96,87 @@ export class SeedingJobCreator {
     });
   }
 
-  private createSeedOptions(mode: SeedMode, geometry: Footprint, cacheName: string): SeedTaskOptions {
+  private handleCleanMode(
+    job: IngestionUpdateFinalizeJob | IngestionSwapUpdateFinalizeJob,
+    cacheName: string,
+    catalogId: string
+  ): { type: string; parameters: SeedTaskParams }[] {
```

Comment:
why an array returned if only a single `{ type: string; parameters: SeedTaskParams }` item is returned